google_metadata_script_runner/main.go

```diff
@@ -428,11 +433,13 @@ func main() {
 	}
 
 	var err error
-	config, err = parseConfig(cfgfile)
-	if err != nil && !os.IsNotExist(err) {
-		fmt.Printf("Error parsing instance config %s: %s\n", cfgfile, err.Error())
+	if err := cfg.Load(nil); err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to load instance configuration: %+v", err)
+		os.Exit(1)
 	}
 
+	config = cfg.Get()
```

Comment:
can we stop using this `config` variable and simply use `cfg.Get()` in the 3 or 4 callers?

Reply:
Done